applications/GeoMechanicsApplication/tests/geo_truss_with_reset_displacement.gid/MeshStructure.svg

```diff
@@ -0,0 +1,296 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
can both distances be placed below the truss? Maybe the force can be placed without overlapping the truss, with its arrow ending on the tip.